### DISCUSSION TITLE
Is it possible to log failed login attempts?

### DISCUSSION THREAD

**Discussion by timaschew** [paperless-ngx/paperless-ngx#1456] [Support] [answered]
I just want to know if anybody tries to brute force my paperless-ngx instance.
Currently nothing is logged, neither logs via docker nor via `/logs` page.

There is some similar post: https://github.com/paperless-ngx/paperless-ngx/discussions/407

But this one would be much more simple and a precondition for the fail2ban

**Comment by shamoon** [comment, marked as answer]
See #2359
